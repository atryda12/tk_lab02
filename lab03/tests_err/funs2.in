
int a = 0, b = 0, c = 0;

int gcd(int m, int n) {

int res = 0;
if (m!=n) { 
    if (m > n)
        res = gcd(m-n);
    else
        res = gcd(n-m, m, m);
}
else
    res = m;

return res;
}

int lcm(int a, int b) {
    string s1="",s2="";
    return (a*b) / gcd(s1,s2);
}

print lcm(125,100);



