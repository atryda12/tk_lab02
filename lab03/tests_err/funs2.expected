
Error: Improper number of args in gcd call: line 9
Error: Improper number of args in gcd call: line 11
Error: Improper type of args in gcd call: line 21
